### ISSUE TITLE
Connect with vendors on auditing standards and norms

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-design-system#2452]
## What

**Timeline:** Estimated from January 2023 to October 2023
**Priority level:** 5 – what makes it easier for service teams to do their jobs

### Potential tasks

- Create a feedback loop between ourselves and agencies which conduct accessibility audits
- Identify issues being raised in audits that affect the GOV.UK Design System

## Epic lead
Not yet assigned

## Why
When service teams hear that the GOV.UK Design System meets WCAG AA criteria, they expect that means that auditing tools and companies won't flag anything from the GOV.UK Design System or GOV.UK Frontend as a WCAG failure. However, sometimes based on research, we implement slightly differently from the norm, in order to increase accessibility.

Setting auditing standards and norms with vendors can help to address and resolve items that are repeatedly flagged in audits. That might include making changes to the design system, or making changes to the way audits are performed. This would save service teams time, effort and grief.

## Who needs to work on this
Internal to team: accessibility specialist, tech leads

External to team: Accessibility Monitoring team

## Who needs to review this
Unknown

## Done when

- [ ] Thing to do

**Comment by davidc-gds** [comment]
Ideas for what we could partner on with vendors:
- Connect on 'issues you've heard about across government'
- Feed back on stuff we're intentionally doing differently, and should be ignored in audits

**Comment by davidc-gds** [comment]
@mgifford great question: our team specifically works with some standards and guidance bodies, but I'm not sure if we're actively or planning to work with any open-source communities. I'll pass the question by the team. 👍
